### DISCUSSION TITLE
Code Format

### DISCUSSION THREAD

**Discussion by BrodieBrown1998** [gh44922023/Tafe230221#7] [Q&A] [answered]
Is this the correct code format to use?
![Code Format](https://github.com/gh44922023/Tafe230221/assets/141377278/e15d0876-9961-4305-8816-e64b5c728355)

**Comment by jeffrey-tafe** [comment, marked as answer]
Looks good to me. I've updated my code to match.

**Comment by jeffrey-tafe** [comment]
@gh44922023 Are you cool with this code/indentation format?

**Reply by gh44922023** [reply]
I have followed suit.
